src/FluentMigrator.Runner.SqlServer/Generators/SqlServer/SqlServer2005Column.cs

```diff
@@ -30,7 +30,7 @@ public SqlServer2005Column(ISqlServerTypeMap typeMap, IQuoter quoter)
         /// <inheritdoc />
         protected override string FormatNullable(ColumnDefinition column)
         {
-            if (column.IsNullable == true && column.Type == null && !string.IsNullOrEmpty(column.CustomType))
+            if (column.IsNullable == true && column.Type == null && !string.IsNullOrEmpty(column.CustomType) && column.Expression == null)
```

Comment:
I really don't understand the purpose of this override to begin with.  Only in the case of using `AsCustom` does it provide `NULL` for a nullable field.  Why for `AsCustom` but not for any other data types?  I can see that perhaps it was desirable to specify `NULL` or `NOT NULL` to override a non-default database option of `ANSI_NULL_DFLT_OFF`, but then why would it be only for `AsCustom` and no other data types?  If this condition were removed, the entire overridden method could be removed.

Reply:
Most likely it is because for all the other types, there are C# structs that map to them and the default is to be non-null.  You have to wrap a struct as Nullable<T> where T is a value type.

I agree, it is weird but would be a breaking change.  If we break it, it would ideally be done in a separate PR and we rebase this PR - this way the PR auto-generates in the release notes as a breaking change and is a lot clearer to end-users.

Is SqlServer2008Column or later re-overriding this behavior though?

Reply:
> Is SqlServer2008Column or later re-overriding this behavior though?

No it isn't.